src/main/java/net/minecraftforge/common/ModDimension.java

```diff
@@ -37,6 +37,21 @@
  */
 public abstract class ModDimension extends ForgeRegistryEntry<ModDimension>
 {
+    /**
+     * Stops the dimension from being saved to level.dat file.
+     * Can be overridden for dynamic dimensions
+     */
+    public boolean saveToLevelDat(){return true;}
+
+
+    /**
+     * Stops the dimension from being loaded from level.dat file.
+     * Override for dynamic dimensions, if you don't want them loaded back from level.dat file
+     * on every server restart.
+     */
+    public boolean loadFromLevelDat(){return true;}
```

Comment:
Is there any specifc reason why you would want to save but not load? If no, just make one function like `persistToLevelDat()` instead of two seperate ones

Reply:
idk any but i thought there might be a reason someone wants that